plugins/metrics/service/src/metricsservice.cpp

```diff
@@ -87,7 +86,7 @@ std::string MetricsServiceHandler::getMetricsFromDir(
       [](const model::File& file) { return file.id; });
 
     if (descendantFids.empty())
-      return "";
+      return;
```

Comment:
Are you sure this change is needed. I tested it on the following code and it gives me an error:
```cpp
#include <iostream>

std::string f() { return; }

int main()
{
  std::cout << f() << std::endl;
  return 0;
}
```
```bash
error: return-statement with no value, in function returning ‘std::__cxx11::string {aka std::__cxx11::basic_string<char>}’ [-fpermissive]
   return;
```

Reply:
You tested the wrong thing. The lambda there is not supposed to return anything.

Reply:
@whisperity All right, I didn't expand the code above and I didn't see that this is in a lambda function.